src/services/request.ts

```diff
@@ -8,52 +8,51 @@ export enum RequestStatus {
   Denied = 'Denied',
 }
 
-export type MembershipRequest = {
-  ActivityCode: string;
-  ActivityDescription: string;
-  /** Comment or text */
+export type RequestUpload = {
+  ACTCode: string;
+  SessCode: string;
+  Username: number;
+  PartCode: Participation;
+  DateSent: string;
   CommentText: string;
+  Status: RequestStatus;
```

Comment:
The naming of these fields seems like a regression. Can we use the full word, instead of abbreviating the `Code` fields?

Reply:
Done